ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -140,6 +140,12 @@ public class ArcticManagementConf {
           .defaultValue(60000L)
           .withDescription("Interval for refreshing table metadata.");
 
+  public static final ConfigOption<Long> GLOBAL_MIN_PLANNING_INTERVAL =
+      ConfigOptions.key("planning.min-interval")
```

Comment:
Is `self-optimizing.min-plan-interval` a better configuration name as we already have a configuration named `self-optimizing.commit-thread-count`. We can group the configurations like below:
```
  self-optimizing:
    commit-thread-count: 10
    min-plan-interval: 60000 # 1min
```

Reply:
BTW, please add the 3 new configurations into `ams/dist/src/main/arctic-bin/conf/config.yaml`.